AppCenterDistribute/AppCenterDistributeTests/MSDistributeConfigureTests.m

```diff
@@ -83,4 +88,69 @@ - (void)testCheckForUpdateDoesNotCheckWhenDisabled {
   [distributeMock stopMocking];
 }
 
+- (void)testCheckForUpdateDoesNotOpenBrowserOrTesterAppAtStartWhenDisabled {
+
+  // If
+  id reachabilityMock = OCMClassMock([MS_Reachability class]);
+  OCMStub([reachabilityMock reachabilityForInternetConnection]).andReturn(reachabilityMock);
+  OCMStub([reachabilityMock currentReachabilityStatus]).andReturn(ReachableViaWiFi);
+  id appCenterMock = OCMClassMock([MSAppCenter class]);
+  OCMStub([appCenterMock isConfigured]).andReturn(YES);
+  id guidedAccessMock = OCMClassMock([MSGuidedAccessUtil class]);
+  OCMStub([guidedAccessMock isGuidedAccessEnabled]).andReturn(NO);
+  OCMStub([appCenterMock isDebuggerAttached]).andReturn(NO);
+  NSDictionary<NSString *, id> *plist = @{@"CFBundleShortVersionString" : @"1.0", @"CFBundleVersion" : @"1"};
+  id utilityMock = OCMClassMock([MSUtility class]);
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wcast-qual"
+  OCMStub(ClassMethod([utilityMock sha256:OCMOCK_ANY])).andReturn(@"RELEASEHASH");
+#pragma GCC diagnostic pop
+  OCMStub([utilityMock currentAppEnvironment]).andReturn(MSEnvironmentOther);
+  id bundleMock = OCMClassMock([NSBundle class]);
+  OCMStub([bundleMock mainBundle]).andReturn(bundleMock);
+  OCMStub([bundleMock infoDictionary]).andReturn(plist);
+  id parserMock = OCMClassMock([MSBasicMachOParser class]);
+  OCMStub([parserMock machOParserForMainBundle]).andReturn(parserMock);
+  OCMStub([parserMock uuid]).andReturn([[NSUUID alloc] initWithUUIDString:@"CD55E7A9-7AD1-4CA6-B722-3D133F487DA9"]);
+  MSDistribute *distribute = [MSDistribute new];
+  id distributeMock = OCMPartialMock(distribute);
+  OCMStub([distributeMock sharedInstance]).andReturn(distributeMock);
+  OCMStub([distributeMock buildTokenRequestURLWithAppSecret:OCMOCK_ANY releaseHash:OCMOCK_ANY isTesterApp:false])
+      .andReturn([NSURL URLWithString:@"https://some_url"]);
+  OCMStub([distributeMock buildTokenRequestURLWithAppSecret:OCMOCK_ANY releaseHash:OCMOCK_ANY isTesterApp:true])
+      .andReturn([NSURL URLWithString:@"some_url://"]);
+  OCMReject([distributeMock openUrlUsingSharedApp:OCMOCK_ANY]);
+  OCMReject([distributeMock openUrlInAuthenticationSessionOrSafari:OCMOCK_ANY]);
```

Comment:
Please add `OCMVerifyAll` calls.

See [9.2 Re-throwing fail fast exceptions in verify all](http://ocmock.org/reference/) for details